hummingbot/strategy/amm_arb/amm_arb.py

```diff
@@ -187,6 +189,21 @@ def apply_budget_constraint(self, arb_proposals: List[ArbProposal]):
                                        f"({balance}) is below required order amount ({required}).")
                     continue
 
+    def prioritize_amm_exchanges(self, arb_proposal: ArbProposal) -> ArbProposal:
+        # TODO: Check if both sides are not part of the EVM_AMM exchanges
+
+        proposal_sides: ArbProposalSide = [arb_proposal.first_side, arb_proposal.second_side]
+
+        active_markets: List[str] = [side.market_info.market.name for side in proposal_sides]
+        if not all([market in AllConnectorSettings.get_gateway_evm_amm_connector_names() for market in active_markets]):
+            return arb_proposal
+
+        results = []
+        for idx, side in enumerate(proposal_sides):
+            pass
+
+        return results
+
```

Comment:
I guess this is still a WIP. Right?

Reply:
Oh yes, I thought i removed it already. Will make the necessary changes